assets/css/_diversion_page.scss

```
@@ -107,3 +107,7 @@
 .c-diversion-panel__help-text {
   line-height: 1.5;
 }
+
+.c-diversion-panel__conclude-detour-text {
+  font-weight: 500;
```

Nit (non-blocking): I'm surprised that we don't already have [a font weight for](https://github.com/mbta/skate/blob/a981349f8d87f62d095c35d0d3c6816b9ddc2a60/assets/css/_skate_ui.scss#L43-L50) medium (if that's what font-weight 500 is called).

If we add it there, then this could become

```suggestion
  font-weight: var(--font-weight-medium);
```

This might conflict with @firestack's comment though. I'm still unsteady on my CSS feet 😅

---

I don't think there's any conflict here! They should coexist well